src/ResidentDirectory.js

```diff
@@ -117,32 +128,35 @@ class ResidentDirectory extends Component {
                     </div>
 
                     {this.state.displayCard &&
-                    <Card style={{width: '30rem', marginTop: "30px", marginLeft: "500px"}}>
-                        <Card.Img variant="top" src="placeHolder.png"/>
+                    <div style={{display: "flex", justifyContent: "center"}}>
+                    <Card style={{width: '30rem', marginTop: "30px"}}>
                         <Card.Body>
-                            <Card.Title>{this.state.value}</Card.Title>
-                            <Card.Subtitle className="mb-2 text-muted">Gender, Age</Card.Subtitle>
-                            <Card.Text>
+                            <Card.Title>
+                                <Card.Img variant="top" src="person.png" style={{height: 30, width:30 }}/>  {this.state.value}
+                            </Card.Title>
+                            <Card.Subtitle className="mb-2 text-muted">{this.state.data.GENDER}, {this.getAge()}</Card.Subtitle>
+                            <div>
                                 <h6>Residence Information:</h6>
                                 <ul style={{listStyleType: "none"}}>
-                                    <li>Current Residence: Village + Room Number</li>
-                                    <li> Entry Date: {this.state.startDate} </li>
-                                    <li>Last Known Residence:</li>
-                                    <li> Previous Shelter Program:</li>
+                                    <li>Current Residence:          {this.state.data.VID}; {this.state.data.RID}</li>
```

Comment:
Would it be more practical to serve the village name and house number instead? I'm not sure how useful a village ID or resident ID would be to an employee.

Reply:
Agreed.

I was working with what we had.
Ideally, we make a call to the Village table and get the village name from the VID.
We can possibly do that for a House as well from the room table.